dags/process_elife_zip_dag.py

```diff
@@ -94,6 +96,19 @@ def prepare_jats_xml_for_libero(**context):
     return etree.tostring(xml)
 
 
+def send_put_request_to_service(**context):
+    return_value = get_return_value_from_previous_task(**context)
```

Comment:
IIRC this means the XML is stored (long-term) in Airflow's database?

Reply:
Correct, return values are stored. For how long? I'm not sure but this is part of Airflow's model.

Reply:
Theoretically you could have tasks running on different workers, so there isn't a shared environment between them where to store data/files apart from S3. So it is conveniente for Airflow to pass the data around. Just curious on whether it's possible to eventually prune the data for old workflows from the database to free space.

Reply:
This is the only documentation I can find about it but I'm going through the XCOM source could reveal its inner workings: https://airflow.apache.org/concepts.html#xcoms